### DISCUSSION TITLE
Error Cloning Repository when running task

### DISCUSSION THREAD

**Discussion by powej2** [semaphoreui/semaphore#952] [General]
I am new to Semaphore and have installed and configured it.

I am using a internal GITLAB server.

I am having difficulty trying to resolve this error:

```
1:42:22 PM Task 4 added to queue
1:42:25 PM Preparing: 4
1:42:25 PM Prepare TaskRunner with template: Add_user
1:42:25 PM Cloning Repository https://gitlab.xxx.com/user/capture-ansible.git
1:42:25 PM Cloning into 'repository_1_1'...
1:42:25 PM fatal: bad numeric config value '0
1:42:25 PM ' for 'GIT_TERMINAL_PROMPT': invalid unit
1:42:25 PM Failed updating repository: exit status 128
```

I could not find any assistance on Google or on GITHUB that seemed to help me resolve this.

**Comment by fiftin** [comment]
Hi @powej2 , sorry for delay,
what OS do you use? What Git version in your system?

**Reply by powej2** [reply]
No worries

I actually do not remember the total issue now as I should have put in more details ;-)

I have the cloning work now so I am good

Thanks for getting back to me.

**Reply by powej2** [reply]
I am new to GIT so I would chalk it up to user error.

**Comment by tboerger** [comment]
I have created the issue https://github.com/ansible-semaphore/semaphore/issues/1806 to track this potential bug.
